cosmos/profiles/__init__.py

```diff
@@ -39,7 +39,6 @@
     DatabricksOauthProfileMapping,
     DuckDBUserPasswordProfileMapping,
     OracleUserPasswordProfileMapping,
-    DuckDBUserPasswordProfileMapping,
```

Comment:
This was a duplicate entry as we already have the same 2 lines up. Docs build reported a duplicate error for this.